### PR TITLE
replace undefined 'escaped' by serializations

### PR DESCRIPTION
**Pull request by therealglazou**
This PR is about https://lists.w3.org/Archives/Public/www-style/2016Mar/0235.html and is ready to merge if the CSSWG agrees on the change later today.

### CODE REVIEW

File: cssom/Overview.bs [hunk 1]
```diff
@@ -2635,12 +2636,12 @@ The <dfn method for=CSS>escape(<var>ident</var>)</dfn> method must return the re
...
 
 <div class=example>
- For example, to escape a string for use as part of a selector, the {{CSS/escape()}} method can be used:
+ For example, to serialize a string for use as part of a selector, the {{CSS/escape()}} method can be used:
  <pre>var element = document.querySelector('#' + CSS.escape(id) + ' > img');</pre>
 </div>
 
 <div class=example>
- The {{CSS/escape()}} method can also be used for escaping strings, although it escapes characters that don't strictly need to be
+ The {{CSS/escape()}} method can also be used for serializing URL fragments, although it escapes characters that don't strictly need to be
```
Comment (zcorpan) [inline cssom/Overview.bs]: Here I meant strings as in CSS strings (as opposed to CSS idents)

Comment (therealglazou) [inline cssom/Overview.bs]: ok, reverting that change then

**Comment by zcorpan** [comment]
LGTM. Do you want to squash?

**Comment by therealglazou** [comment]
@zcorpan done

**Comment by zcorpan** [comment]
Thanks, merged as https://github.com/w3c/csswg-drafts/commit/147461bd13ebc8ad342a9532a2c3eb03d0e57626

**Pull request closed without merging**